### ISSUE TITLE
Riddle School 3, Game Crashes

### ISSUE THREAD

**Issue by thenickman100** [ruffle-rs/ruffle#19036]
### Describe what you were doing

After beating Riddle School 3, the game crashes during the credits
![Image](https://github.com/user-attachments/assets/92000860-6482-4c2b-8d22-0a917c9d67f4)

### What does the crash message say?

This error message is repeated 3000 times:
ruffle_web-wasm_extensions.js:1258 ERROR web/src/lib.rs:1010 Unable to lock Ruffle core

### Ruffle Version

29 Nov Build

### Affected platform

Self-hosted version

### Operating system

Windows 11

**Comment by Aaron1011** [comment]
Can you upload a save file where you've beaten the game?

**Comment by thenickman100** [comment]
> Can you upload a save file where you've beaten the game?

Yes, but how do I do this?  I assume I'll have to play through the game again, right?
